src/js/camresolutionstest.js

```diff
@@ -175,16 +175,16 @@ CamResolutionsTest.prototype = {
     var statsReport = {};
     var frameStats = frameChecker.frameStats;
 
-    for (var index = 0; index < stats.length - 1; index++) {
+    for (var index in stats) {
```

Comment:
I am not sure on the recomended way to loop through elements on a dictionary. If you would have time to look that would be great, but mostly i am afraid this would loop through methods of the dictionary if there was a library that injects them on the page.